app/helpers/course/controller_helper.rb

```diff
@@ -46,4 +46,30 @@ def link_to_user(user, options = {}, &block)
       super(user, options, &block)
     end
   end
+
+  # Display the course_user_badge given the course and course_user
+  #
+  # @return [String] HTML string to render the course_user_badge
+  def display_course_user_badge
+    exp = current_course_user.experience_points
```

Comment:
But `nil.staff?` will still give exceptions ?

Yep, I think that make sense. and it's consistent with our user helpers: `display_user(user)`